templates/docs/shared/_footer.html

```diff
@@ -0,0 +1,40 @@
+<div class="p-strip is-dark l-docs__footer">
+  <div class="row">
+    <footer class="l-docs__subgrid">
+      <div class="l-docs__sidebar">
+        <p style="padding-left: 1.5rem">
+          Copyright &copy; {{ now("%Y") }}
+          <br />
+          CC-BY-SA 3.0, Canonical Ltd.
+        </p>
+      </div>
+      <div class="l-docs__main">
+        <nav aria-label="Footer">
+          <div class="has-cookie">
+            <ul class="p-inline-list--middot">
+              <li class="p-inline-list__item">
+                <a aria-label="External link to the Ubuntu legal information page"
+                   href="https://www.ubuntu.com/legal">Legal information</a>
+              </li>
+              <li class="p-inline-list__item">
+                <a class="js-revoke-cookie-manager" href="">Manage your tracker settings</a>
```

Comment:
```suggestion
                <a class="js-revoke-cookie-manager" href="#">Manage your tracker settings</a>
```
If JS is disabled this will refresh the page, it's better that it just does nothing

Reply:
Sure I can add that, although this is how it was in multipass.run: https://github.com/canonical/multipass.run/blob/4e54511c235648a61b67d1499af21e98d53bf19a/templates/_layouts/_footer.html#L20C13-L20C89

Reply:
I suppose multipass was also wrong